crates/bevy_ecs/src/component/mod.rs

```diff
@@ -340,8 +340,19 @@ impl ComponentTicks {
         check_tick(&mut self.changed, change_tick);
     }
 
+    /// Manually sets the change tick.
+    /// Usually, this is done automatically via the [`Deref`](std::ops::Deref) implementation on [`Mut`](crate::world::Mut) or [`ResMut`](crate::system::ResMut) etc.
+    ///
+    /// # Example
+    /// ```rust,no_run
+    /// # use bevy_ecs::{world::World, component::ComponentTicks};
+    /// let world: World = todo!();
+    /// let component_ticks: ComponentTicks = todo!();
```

Comment:
Do you intend to fill this in this PR?

Reply:
I just wanted the variables in scope, but I agree that `todo!()` doesn't really fit. Do you think changing it to `unimplemented!()` would be fine or would you like a larger example with proper values?

Its hard to come up with a simple example as this is the code I used (copied from bevy):
```rust
#[inline]
unsafe fn get_component_and_ticks(
    world: &World,
    component_id: ComponentId,
    entity: Entity,
    location: EntityLocation,
) -> Option<(*mut u8, *mut ComponentTicks)> {
    let archetype = &world.archetypes()[location.archetype_id];
    let component_info = world.components().get_info_unchecked(component_id);
    match component_info.storage_type() {
        StorageType::Table => {
            let table = &world.storages().tables[archetype.table_id()];
            let components = table.get_column(component_id)?;
            let table_row = archetype.entity_table_row(location.index);
            // SAFE: archetypes only store valid table_rows and the stored component type is T
            Some((
                components.get_unchecked(table_row),
                components.get_ticks_unchecked(table_row),
            ))
        }
        StorageType::SparseSet => world
            .storages()
            .sparse_sets
            .get(component_id)
            .and_then(|sparse_set| sparse_set.get_with_ticks(entity)),
    }
}
```

Reply:
Yeah lets just change these to unimplemented().

Reply:
Done